sbnana/SBNAna/NuMINumuXSec/ICARUSNumuXsec_Cuts.h

```diff
@@ -0,0 +1,112 @@
+#ifndef ICARUSNumuXsec_Cuts_h
+#define ICARUSNumuXsec_Cuts_h
+
+#include "sbnana/SBNAna/NuMINumuXSec/ICARUSNumuXsec_Variables.h"
+
+namespace ICARUSNumuXsec{
+
+  //==== NC
+  const Cut cutIsNuMuNC([](const caf::SRSliceProxy* slc) {
+      return ( kIsNuSlice(slc) && slc->truth.isnc && ( slc->truth.pdg == 14 || slc->truth.pdg == -14 ) );
+    });
+
+  const Cut cutTruthNoPiZero([](const caf::SRSliceProxy *slc){
+    return (varTruthNPiZero(slc)==0);
+  });
+
+  const Cut cutTruthNoChargedPion([](const caf::SRSliceProxy *slc){
+    return (varTruthNChargedPion(slc)==0);
+  });
+
+  const Cut cutTruthNoNeutron([](const caf::SRSliceProxy *slc){
+    return (varTruthNNeutron(slc)==0);
+  });
+
+  //==== FV
+
+  const Cut cutRFiducial([](const caf::SRSliceProxy* slc) {
+
+    double XMargin = 25.;
+    double YMargin = 25.;
+    double ZMarginUp = 30.;
+    double ZMarginDown = 50.;
+
+    return ( !isnan(slc->vertex.x) &&
+           ( ( slc->vertex.x < -71.1 - XMargin && slc->vertex.x > -369.33 + XMargin ) ||
+           ( slc->vertex.x > 71.1 + XMargin && slc->vertex.x < 369.33 - XMargin ) ) &&
+           !isnan(slc->vertex.y) &&
+           ( slc->vertex.y > -181.7 + YMargin && slc->vertex.y < 134.8 - YMargin ) &&
+           !isnan(slc->vertex.z) &&
+           ( slc->vertex.z > -895.95 + ZMarginUp && slc->vertex.z < 895.95 - ZMarginDown ) );
+  });
+
+  //==== FMScore
+  const Cut cutFMScore([](const caf::SRSliceProxy* slc) {
+      return ( !isnan(slc->fmatch.score) && slc->fmatch.score < 6.0 );
+    });
+
+  const Cut cutHasMuon([](const caf::SRSliceProxy* slc) {
+    return (varMuonTrackInd(slc) >= 0);
+  });
+
+  const Cut cutMuonContained([](const caf::SRSliceProxy* slc) {
+
+    bool Contained(false);
+    if( varMuonTrackInd(slc) >= 0 ){
+      auto const& trk = slc->reco.trk.at(varMuonTrackInd(slc));
+      Contained = ( !isnan(trk.end.x) &&
+                  ( trk.end.x < -71.1 - 25 && trk.end.x > -369.33 + 25 ) &&
+                  !isnan(trk.end.y) &&
+                  ( trk.end.y > -181.7 + 25 && trk.end.y < 134.8 - 25 ) &&
+                  !isnan(trk.end.z) &&
+                  ( trk.end.z > -895.95 + 30 && trk.end.z < 895.95 - 50 ) );
+      return Contained;
+    }
+    return Contained;
+
+  });
+
+
+  const Cut cutMuonMatchedToMuon([](const caf::SRSliceProxy* slc) {
+    return ( abs(varMuonTruePDG(slc)) == 13 );
```

Comment:
Unfortunately, I don't think I set it up so you can call `abs()` directly on Vars, but that is something I could consider adding.

Reply:
Does it mean that I have to something like 
```suggestion
    return ( abs(varMuonTruePDG(slc)) == 13 );
```

Reply:
Does this mean I have to do something like, 
```suggestion
    int pdg = varMuonTruePDG(slc);
    return ( abs(pdg)==13 ); 
```

Reply:
(at least for now)

Reply:
Ah, maybe what you meant is currently `abs(varMuonTruePDG)==13` is not supported? (without `slc`)?

Reply:
Right. What you have is fine, but it would be nice to just write `const Cut cutMuonMatchedToMuon = abs(varMuonTruePDG)==13;`, but I don't think that would work at the moment.